tests/vmi_lifecycle_test.go

```diff
@@ -1549,7 +1545,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 		})
 		Context("with ACPI and some grace period seconds", func() {
 			DescribeTable("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should result in vmi status succeeded", func(gracePeriod int64) {
-				vmi = newCirrosVMI()
+				vmi = libvmi.NewCirros()
```

Comment:
Maybe you could also include the gracePeriod option.

Reply:
done